.github/workflows/publish-to-pypi-testpypi.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish distributions to PyPI (releases only) and TestPyPI (all commits)
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish distributions to PyPI (releases only) and TestPyPI (all commits)
+    if: github.repository == 'pvlib/pvlib-python'
+    runs-on: ubuntu-latest
+    steps:
+    # fetch all commits and tags so versioneer works
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Fetching all of history seems a little overkill -- is it worth it to copy [metpy's approach](https://github.com/Unidata/MetPy/blob/master/.github/workflows/release.yml#L17)?  Or maybe fetching is cheap so let's not worry about it.

Reply:
It doesn't take more than a few seconds for this to run so I think we should keep it simple for now.